### PR TITLE
General refactoring for youtubenlp

### PR DESCRIPTION
**Pull request by pandevim**
## Description

- Little tweaks and directory structure refactoring. 
- Added [AUTHORS](https://github.com/YouTubeNLP/Backend/blob/refactor/directory-structure/AUTHORS) detail.
- Fixes build problems from #19 which happend due to wrong merge conflict resolve made by @pandevim.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested? (Optional)

Manual testing (PASSED)
image
Automated [tests](https://github.com/YouTubeNLP/Backend/runs/961772810) are failing. See #26 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Aniruddha Pandey <anirudh.pandev@gmail.com>
+Edward Huang <edwhuang@umich.edu>
+Pratik Karki <karkipra@grinnell.edu>
```
Comment (karkipra) [inline AUTHORS:3]: Do you mind changing this to karkip@gmail.com?

**Comment by karkipra** [comment]
Seems good except for the failing tests. I guess we just have to wait for it to be fixed.

**Comment by pandevim** [comment]
yep waiting on #26 🤞

**Comment by pandevim** [comment]
Merging this quickly so that people can star working with the update code.

**Pull request merged**